now.json

```diff
@@ -1,10 +1,26 @@
 {
-  "version": 1,
-  "type": "docker",
+  "version": 2,
   "alias": [
-    "badgen.net",
-    "flat.badgen.net"
+    "v2.badgen.net"
   ],
+  "regions": ["sfo", "lhr", "hnd"],
+  "builds": [
+    { "src": "api/*.ts", "use": "@now/node" },
+    { "src": "package.json", "use": "@now/static-build" }
+ ],
+  "routes": [
+    { "src": "/", "headers": { "cache-control": "public, maxage=604800, stale-while-revalidate: 86400" } },
+    { "src": "/builder", "headers": { "cache-control": "public, maxage=604800, stale-while-revalidate: 86400" } },
+    { "src": "/(?<name>[^/]+).*", "dest": "/api/$name.ts" }
+  ],
+  "env": {
+    "GH_TOKENS": "@badgen-gh-tokens",
+    "SENTRY_DSN": "@badgen-sentry-dsn",
+    "TRACKING_GA": "@badgen-tracking-ga"
+  },
+  "github": {
+    "autoAlias": false
+  },
   "files": [
```

Comment:
I don't believe `files` property is supported for v2. You'll need a `.nowignore` file instead.

Reply:
Nice, cleaner `now.json` ❤️